installers/metrics/ansible/values.yaml

```diff
@@ -3,6 +3,17 @@
 # More info for these options can be found in the docs
 # https://access.crunchydata.com/documentation/postgres-operator/latest/installation/metrics/metrics-configuration/
 # =====================
+alertmanager_custom_configmap: ""
+alertmanager_custom_rules_configmap: ""
+alertmanager_install: "true"
+alertmanager_log_level: "info"
+alertmanager_port: "9093"
+alertmanager_service_name: "crunchy-alertmanager"
+alertmanager_service_type: "ClusterIP"
+alertmanager_storage_access_mode: "ReadWriteOnce"
+alertmanager_storage_class_name: "fast"
+alertmanager_supplemental_groups: "65534"
+alertmanager_volume_size: "1G"
```

Comment:
I would opt to use `Gi` instead of `G`

Reply:
Sure, I'll update it here and for prometheus and grafana unless i hear otherwise.